resources/views/app/authlayout.blade.php

```diff
@@ -56,6 +57,10 @@
         <!-- Password Reset-->
         <script src = "{!! asset('/js/PasswordReset.js') !!}"></script>
         <script src = "{!! asset('/js/NewPassword.js') !!}"></script>
```

Comment:
PasswordReset.js is taking care of the first stage of the process. And NewPassword.js is for the second process, there was token mismatch because NewPassword.js takes in 2 token.